hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -255,6 +260,20 @@ public boolean getIsNamespaceEnabled() throws AzureBlobFileSystemException {
     return isNamespaceEnabled;
   }
 
+  @VisibleForTesting
+  boolean isNameSpaceEnabledSetFromConfig() {
+    final String hnsEnabledConfig = abfsConfiguration
+        .getIsNamespaceEnabledAccount();
```

Comment:
is line break needed?

Reply:
Yes this is needed.
![image](https://user-images.githubusercontent.com/52483117/79967563-9c50be00-84ac-11ea-8f8d-d020a433f133.png)